### ISSUE TITLE
New project name?

### ISSUE THREAD

**Issue by gunnarx** [MEAE-GOT/W3C_VehicleSignalInterfaceImpl#147]
- This is the most feature complete implementation of VISS v2
- It is in effect a kind of reference implementation for the VISS specification development (but maybe not "officially")
- It is frequently referenced by various projects and discussions.

But it is not the only implementation.  At least these exist:
 - https://github.com/aShinjiroUrata/vehicle-information-service-spec
 - and another one in Kuksa
(they are both VISS v1 /  WebSocket based).

Therefore I wonder if this project could get an official and more convenient name? The name of this repo is not the easiest to refer to, and using the company name to clarify the implementation seems inappropriate because there is more development by Geotab than by Melco now.

I would not propose a very advanced or creative name, but something that is easy to refer to and indicates what it is.

What do you think?

**Comment by MagnusGun** [comment]
@gunnarx yes it should probably change its name since it has moved away from being a testing ground for version 2 of the viss spec, to be as you say an reference implementation...

@UlfBj @petervolvowinz what do you think about WISE (W3c automotIve acceSs implEmentation) ??
https://acronymify.com/search?q=w3c+automotive+access+implementation

**Comment by UlfBj** [comment]
With such wise men involved, not so bad.

Alternatives: 
WAIR/WAIRE (w3c automotive interface reference)
WAII (w3c automotive interface implementation)  pronounced wy-ee, cmp hawaii

**Comment by petervolvowinz** [comment]
Go for it.
